docs/source/en/intro/quickstart.md

```diff
@@ -408,6 +408,7 @@ with the above short sections.
 We recommend that developers continue reading other documents:
 - Egg provides a powerful mechanism for extending features. See [Plug-ins](../advanced/plugin.md).
 - As large teams need to follow certain constraints and conventions, in egg, we recommend further creating an upper-level framework that fits your team. See [Frameworks](../advanced/framework.md).
```

Comment:
Egg framework allows small or large teams to work together as fast as possible under the well-documented conventions and coding best practices. In addition, the teams can build up logics on top of the framework to better suited their special needs. See more on [Frameworks](../advanced/framework.md).